types/node/test.d.ts

```diff
@@ -1223,6 +1223,11 @@ interface TestFail {
          * The error thrown by the test.
          */
         error: Error;
+        /**
+         * The type of the test, used to denote whether this is a suite.
+         * @since 20.0.0, 19.9.0, 18.17.0
+         */
+        type?: 'suite';
```

Comment:
Yep, it's "suite" or it's not present. I've been working with the test runner codebase and can confirm it does not get set to something else. I guess it could in the future, but not today.